cups/ppd-private.h

```diff
@@ -131,6 +141,11 @@ struct _ppd_cache_s			/**** PPD cache and PWG conversion data ****/
 					/* Number of print-color-mode/print-quality options */
   cups_option_t	*presets[_PWG_PRINT_COLOR_MODE_MAX][_PWG_PRINT_QUALITY_MAX];
 					/* print-color-mode/print-quality options */
+  int		num_optimize_presets[_PWG_PRINT_CONTENT_OPTIMIZE_MAX];
```

Comment:
They aren't presets, they are print-content-optimize values.

Reply:
Yes, I have "invented" presets for these, to implement the IPP-attribute->Set-of-PPD-option-settings assignment. One could name them differently. My intuition was here that a "preset" is simply nothing more than as set of PPD option settings, key/value pairs, independent of by which IPP attribute(s) the preset gets selected.

Reply:
If you are mapping print-content-optimize, then please call it a mapping (like I do for everything else). Presets have a very specific meaning in IPP and mixing terminology just makes things more confusing.

Reply:
OK, will do.

Reply:
Renamed everything appropriately now.